functions.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+function displayCollection(array $allCards) : string {
+    $collection = '';
+    foreach ($allCards as $card) {
+        $collection .=
+            '<span>________________________________________</span>' .
```

Comment:
this should be achieved using css instead (or removed)

Reply:
fixed